app/services/hyrax/file_set_fixity_check_service.rb

```diff
@@ -12,91 +48,54 @@ def initialize(file_set)
       end
     end
 
-    NO_RUNS = 999
+    # Fixity checks each version of each file if it hasn't been checked recently
+    # If object async_jobs is false, will returns the set of most recent fixity check
+    # status for each version of the content file(s). As a hash keyed by file_id,
+    # values arrays of possibly multiple version checks.
+    #
+    # If async_jobs is true (default), just returns nil, stuff is still going on.
+    def fixity_check
+      results = file_set.files.collect { |f| fixity_check_file(f) }
 
-    # provides a human readable version of the fixity check status
-    # This may trigger fixity checks if required
-    # @param [Hydra::PCDM::File] file the file to get the fixity check status for,
-    #                                 defaults to the original_file.
-    def human_readable_fixity_check_status(file = file_set.original_file)
-      fixity_check_stat(file)
-    end
+      return if async_jobs
 
-    # Check the file by only what is in the fixity check log.
-    # Do not try to access the versions if we do not have access to them.
-    # Use this when a file_set is loaded from solr instead of fedora
-    def logged_fixity_status
-      fixity_results = ChecksumAuditLog.logs_for(id, "original_file")
-                                       .collect { |result| result["pass"] }
-
-      if !fixity_results.empty?
-        stat_to_string(fixity_results.reduce(true) { |sum, value| sum && value })
-      else
-        'Fixity checks have not yet been run on this file.'
-      end
+      results.flatten.group_by(&:file_id)
     end
 
-    # Fixity checks each version of each file if it hasn't been checked recently
-    # Returns the set of most recent fixity check status for each version of the
-    # content file
-    # @param [Hash] log container for messages, mapping file ids to status
-    def fixity_check(log = {})
```

Comment:
Thus method hasn't actually been removed, the git diff has just separated it from where it remains since there were so many changes to this file. It does now take no params instead of a `log` param of unclear utility that was not being used internally in hyrax.